include/aos/iam/certhandler.hpp

```diff
@@ -26,6 +24,55 @@ namespace certhandler {
  */
 constexpr auto cIAMCertModulesMaxCount = AOS_CONFIG_CERTHANDLER_MODULES_MAX_COUNT;
 
+/**
+ * Max number of certificate subscriptions:
+ *
+ * 1 per client & server + cIAMCertModulesMaxCount for message proxy
+ */
+constexpr auto cIAMCertSubsMaxCount = cIAMCertModulesMaxCount * 2 + cIAMCertModulesMaxCount;
```

Comment:
It should be private of CertHandler const. Also unclear why we need +cIAMCertModulesMaxCount

Reply:
1 per client & server + cIAMCertModulesMaxCount for message proxy